api/IntersectionObserver.json

```diff
@@ -0,0 +1,52 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "api": {
+      "IntersectionObserver": {
+        "__compat": {
+          "basic_support": {
+            "support": {
+              "chrome": {
+                "version_added": "51"
+              },
+              "edge": {
+                "version_added": true
+              },
+              "firefox": [
```

Comment:
Makes sense. I'll fix it. I've removed the bug reference too, although we've historically included bug references in notes about things being preffed off, so I included it. As for the bug number, that's the bug the ddn was on, so it's the one I referenced. :)